readme.md

```diff
@@ -78,7 +78,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Question:** What are query parameters? Try going to a couple of your favorite websites and paste an example of query parameters being used.
 
-* **Your Answer:**
+* **Your Answer:anything after ? you have a key value pair.  for about course is the key and javascript is the value
```

Comment:
Passes these additional parameters in the url after the route. Useful for filters etc.